### DISCUSSION TITLE
When can we solve the problem of the slow running of the farmer interface beyond 1pb?

### DISCUSSION THREAD

**Discussion by NTOHPN** [Chia-Network/chia-blockchain#11437] [Support]
When the farmer's capacity is greater than 1pb, the GUI interface runs slowly.

**Comment by BigSilverOrb** [comment]
Hear hear! the recent merge did not address.

**Comment by FuzeGuy** [comment]
The farmer works ok doing its thing, it's just the interface is slow to respond ... at even ~.5PB size. Not really a big issue, just annoying at times.

**Reply by NTOHPN** [reply]
需要官方优化一下
